invenio_app_ils/api.py

```diff
@@ -53,4 +54,4 @@ def item_exists(item_pid):
 
 def patron_exists(patron_pid):
     """Return True if the Patron exists given a PID."""
-    return True
+    return User.query.filter_by(id=patron_pid).first() is not None
```

Comment:
Should we refer to the patron ID as a PID even if it's not a record?

Reply:
Agree with you, the param name comes from `invenio-circulation` that does not know if patron is a record or not. Given that, I would probably not change the "signature" of the method. WDYT?